### ISSUE TITLE
Feature Request: Ability for paper-slider to be reversed

### ISSUE THREAD

**Issue by bradsk88** [PolymerElements/paper-slider#129]
I have an application where I need the _left side_ of the slider to have the value "500" and the _right side_ to have the value "150".
### Actual outcome

If I attempt to create a `paper-slider` with the `min=500` and `max=150`

```
<paper-slider max="100" min="500"></paper-slider>
```

... I get the following error:

```
paper-slider.html:660 Uncaught RangeError: Invalid array length
```

I did some digging and I know that the problem runs deep.  A simple change at `paper-slider.html:660` will not be enough.
### Expected outcome

Specifying inverted min and max values should be allowed.
Clicking (for example) at the 25% mark between a `min=100` and a `max=0` should result in a `value=75`

-- OR --

Specifying a `reversed` boolean on the `paper-slider` should result in the same behaviour, except the `min` value will remain the smaller of the two.

### Live Demo

https://jsbin.com/qobaluzevu/edit?html,output
### Steps to reproduce
1. Put a `paper-slider` element in the page having a `min` value greater than the `max` value.
2. Open the page in a web browser.
3. Check the browser console to see the error
### Browsers Affected

- [x] Chrome
- [x] Firefox
- [x] Safari 9
- [x] Safari 8
- [x] Safari 7
- [x] Edge
- [x] IE 11
- [x] IE 10

**Comment by keanulee** [comment]
Hi @bradsk88 

That's an interesting feature request, but I think having reversed values is a pretty niche use case to be implemented at the element level. Have you considered using paper-slider as intended, then transforming its value to what you need (e.g. `return 500 - paperSlider.value;`)

**Comment by bradsk88** [comment]
Yes.  I suppose I could set the paper-slider to have min=0 and max=100 and just map the values to my range.

If you don't decide to support inverted ranges, having the component fail fast with a clear error message would probably be the next best alternative.
